pkg/build/builder.go

```diff
@@ -303,16 +303,17 @@ func exist(paths ...string) error {
 type BuilderName string
 
 const (
-	BuilderNameGo     BuilderName = "go"
-	BuilderNameDeno   BuilderName = "deno"
-	BuilderNamePython BuilderName = "python"
-	BuilderNameNode   BuilderName = "node"
-	BuilderNameDocker BuilderName = "docker"
+	BuilderNameGo          BuilderName = "go"
```

Comment:
```suggestion
	NameGo          Name = "go"
```

(https://github.com/golang/go/wiki/CodeReviewComments#package-names)

Reply:
Yeah TBH this can just merge into TaskKind, but I'll rename it for now